### DISCUSSION TITLE
Google Calendar connect not available

### DISCUSSION THREAD

**Discussion by Dhruviika** [calcom/docker#249] [General]
When I installed the cal.com repo in my local system. It worked fine but I am unable to login via google calendar it just shows options like APPLE CAL , CALDEV etc

**Comment by wokawoka** [comment]
I'm experiencing the same issue

**Reply by Dhruviika** [reply]
Yeah, so you try this
Close everything and re-start the project again then like in my case you would be able to see "skip" option and then you can login easily. But still Google Calendar option is not there.

**Comment by krumware** [comment]
Can you please provide a copy of your env vars (de-identified)?
